app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parameters = command.split()
+
+    if (len(parameters) == 3
+            and parameters[0] == "cp"
+            and parameters[1] != parameters[2]):
+        new_file = parameters[2]
+        old_file = parameters[1]
```

Comment:
```
new_file = parameters[2]
old_file = parameters[1]
```

add this code before "if" condition to not duplicate calls parameters[1] and parameters[2]